### DISCUSSION TITLE
Hi api-platform team just wondering if one of you has plans using api-platform with Joomla Web Services?

### DISCUSSION THREAD

**Discussion by alexandreelise** [api-platform/api-platform#2559] [Q&A]
Hi Kévin @dunglas,

I'm Alex. I recently listened again your podcast episode on API Intersection and you said that you would like more projects and languages outside of just php and symfony using and contributing back to api-platform rather than reinvent the wheel. I wanted to start creating a tool for Joomla Web Services (joomla 5.0.0) the CMS named routefinder that would extract Web Services endpoints back to an OpenAPI 3.0.0 and later multi-file definition. Usually it's the other other way. But I consider using api-platform instead for this use case. Am I right? Does it fit this use-case or should I continue working on my own tool. Thanks for your time and patience. Continue your great work you, your team and all api-platform contributors.

**Comment by soyuka** [comment]
Yes definitely fits API Platform has lots of things working on OpenAPI specification. You want to introduce api-platform to Joomla Web Service to generate an OpenAPI documentation? 
You'll definitely find many tools you may need here, feel free to reach me on the symfony slack if you want to discuss further.
